test/transactions.go

```diff
@@ -12,6 +16,7 @@ func TransferTransaction() *transferTransaction {
 			Transaction: &protocol.TransactionBuilder{
 				ContractName: "BenchmarkToken",
 				MethodName:   "transfer",
+				Timestamp:    primitives.Timestamp(time.Now().UnixNano()),
```

Comment:
see orbs-network/orbs-spec/issues/49, the name of the field does not reflect its meaning. What does the timestamp represent? date it was created? added to the system? in which timezone? the end user's? ours? utc?

Reply:
Already asked @OdedWx to change to TimestampNano

Reply:
I don't think this is good enough. I'd like the field name to reflect the meaning as well as the units. For instance: `dateCreatedInNanos`. 

BTW, we (@jlevison, @netoneko and myself) assumed this field was in millis!